src/database/schema.js

```diff
@@ -214,7 +214,7 @@ export const schema = {
     VaccineVialMonitorStatus,
     VaccineVialMonitorStatusLog,
   ],
-  schemaVersion: 19,
+  schemaVersion: 20,
```

Comment:
Master is currently `18` so I think it's best not to bump this past `19` for this release?